roles/tas_single_node/tasks/volumes.yml

```diff
@@ -1,13 +1,17 @@
 ---
 - name: Create volume list variable
   ansible.builtin.set_fact:
-    volume_list:
-      - trillian-mysql
-      - rekor-server
-      - rekor-redis-storage
-      - redis-backfill-storage
-      - "{{ tas_single_node_tuf_repository_volume_name }}"
-      - "{{ tas_single_node_tuf_signing_keys_volume_name }}"
+    volume_list: >-
+      {{
+        (["trillian-mysql"] if tas_single_node_trillian.database_deploy else []) +
+        [
+          "rekor-server",
+          "rekor-redis-storage",
```

Comment:
I think using external Redis is also an option ATM, isn't it? Meaning that we would also have to conditionalize this (`tas_single_node_rekor_redis.database_deploy` can be used for the condition)